python/ray/air/tests/execution/test_e2e_train_flow.py

```diff
@@ -205,8 +205,7 @@ def training_barrier_completed(self, barrier: Barrier):
             self._actor_manager.remove_actor(tracked_actor)
 
     def training_error(self, tracked_actor: TrackedActor, exception: Exception):
```

Comment:
When a training error occurs, do we need to restart the other actors/current barrier?

Reply:
No, the other actors are fine. In a real setting they may want to restart their training loop or we can restart the actors. This is just a proof of concept to show that a train-like flow can be handled using the execution engine and its abstractions.